Update.md

```diff
@@ -0,0 +1,79 @@
+# How to update code examples in KILT workshop
+
+Please see [here](https://github.com/KILTprotocol/organizational/wiki/How-to-update-code-example-in-the-KILT-workshop#2-apply-changes-from-workshop-to-sdk). To be aware of breaking changes to the workshop code snippets, we execute each snippet in the SDK CI pipeline `test_workshop`.
+
```

Comment:
if you're pulling the tests into this repo, the references to the sdk will have to be changed, but most of these instructions will probably stay as they are.